bin/setup

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+git submodule init; git submodule update --init
```

Comment:
I think we do `git submodule update --init` to avoid calling `git submodule init` first. Aren't these two commands redundant?